R/kmeans-clustering-method.R

```diff
@@ -0,0 +1,355 @@
+#' Build clusters using kmeans()
+#'
+#' @description This step allows you to use kmeans() to build clusters, based on
+#' your data.
+#' @docType class
+#' @usage KmeansClustering(object, df, grainCol, labelCol, numOfClusters,
+#' usePrinComp, numOfPrinComp,impute, debug)
+#' @importFrom R6 R6Class
+#' @import ranger
+#' @param object of UnsuperviseModelParameters class for $new() constructor
+#' @param df Dataframe whose columns are used for calc.
+#' @param grainCol Optional. The dataframe's column that has IDs pertaining to 
```

Comment:
This is a lot of params for analysts to have to sort out. Let's make sure there are simple examples that only use the required params (so folks aren't intimidated).